engine/apps/auth_token/auth.py

```diff
@@ -46,6 +47,10 @@ def authenticate_credentials(self, token):
             auth_token = self.model.validate_token_string(token)
         except InvalidToken:
             raise exceptions.AuthenticationFailed("Invalid token.")
+
+        if auth_token.organization.migration_destination is not None:
```

Comment:
It looks like this is already populated before we get here by this point:
https://github.com/grafana/oncall/blob/5a4c3d6d7716b7b06e43bb56347ce96f9948da44/engine/apps/auth_token/models/base_auth_token.py#L36

Does it make sense to change anything there?

Reply:
I was referring to the `migration_destination` FK to the region; if you are only interested in knowing if there is a related object, checking for the id field will not trigger a db query (as described here: https://stackoverflow.com/questions/46163890/does-looking-at-a-django-models-fk-id-attribute-trigger-a-database-lookup); although since when there is a value you then use it in the middleware, probably it's the same in this case (ie. the query to fetch the region is needed/used when the FK is set)

Reply:
What was confusing me here was setting db_column on the ForeignKey does not change that Django field still has a _id suffix in the code. All of it makes sense now.